### DISCUSSION TITLE
How do I access cf object on the context?

### DISCUSSION THREAD

**Discussion by vinaymhetre** [honojs/hono#961] [General]
I have just started a new project with hono 3.0.2 using create-hono CLI.

I could previously access the cloudflare object and its properties in workers project with Hono but they seem to be not available anymore. Any idea where to look for these for the incoming request?

`app.get('/', async (c) => {
  const country = c.req.cf?.country;
  return c.text(`Hello from ${country}`);
});`

**Comment by yusukebe** [comment]
You can access it from `c.req.raw.cf`.

**Reply by vinaymhetre** [reply]
Great, thanks a lot!
